src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/OverseasEntitySubmissionDto.java

```diff
@@ -259,14 +259,7 @@ public String getWhoIsRegistering() {
         return whoIsRegistering;
     }
 
-    public void setWhoIsRegistering(String whoIsRegistering) {
-        if (whoIsRegistering != null) {
```

Comment:
Looks like code used to convert the supplied value to uppercase but no longer. Wondering why this was done and why it's being removed - there's no mention of this in the JIRA ticket.

Reply:
The code that was here has been moved into the validator.